dev/com.ibm.websphere.javaee.el.3.0/src/javax/el/ImportHandler.java

```diff
@@ -15,24 +15,267 @@
  * limitations under the License.
  */
 // wtlucy: Apache bug 57142 patch pulled in - file rev 1644523
+// epj: backfit EL 4.0 perf improvement
```

Comment:
If we have a tomcat bug we're pulling in would be good to detail that here as well as Bill did in his edit above yours.

Reply:
updated comment to mention apache bug